lib/system/dollars.nim

```diff
@@ -1,8 +1,25 @@
+include system/inclrtl
+
 proc `$`*(x: int): string {.magic: "IntToStr", noSideEffect.}
   ## The stringify operator for an integer argument. Returns `x`
   ## converted to a decimal string. ``$`` is Nim's general way of
   ## spelling `toString`:idx:.
 
+when defined(js):
+  when (NimMajor, NimMinor, NimPatch) >= (1, 3, 3):
```

Comment:
Shouldn't this be `>= (1, 3, 2)`?

Reply:
i thought about this but:
* see https://github.com/nim-lang/Nim/blob/af9c85270198455c7ba218fae919e79060a8960a/compiler/docgen.nim#L736 which uses NimPatch as indicator
* it reserves a possibility to have "unstable releases" on `1.(2x+1).(2y)` branches which could be a neat idea

Reply:
What exactly is Nim v1.3.3, @timotheecour?

(Spoiler: it doesn't exist nor will it exist)

Reply:
that was already reverted from this PR and is discussed elsewhere